src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -81,7 +73,31 @@ protected void runTool(final JavaSparkContext ctx) {
             }
 
         });
-        assembledBreakpoints.saveAsTextFile(output);
+        allContigAlignments.saveAsTextFile(output);
+    }
+
+    /**
+     * Loads an RDD of {@link ContigsCollection} objects keyed by assembly ID from disk. The input file
+     * should be the output of as RunSGAViaProcessBuilderOnSpark.
+     */
+    static JavaPairRDD<String, ContigsCollection> loadContigsCollectionKeyedByAssemblyId(final JavaSparkContext ctx, final String inputPath) {
+        final JavaRDD<String> inputAssemblies = ctx.textFile(inputPath).cache();
```

Comment:
Thanks for catching that, the cache() was part of an earlier experiment that became obsolete when I reorganized the code. I've removed it.

But my understanding is that Spark will automatically unpersist RDDs from the cache as needed so you don't always need to call `unpersist()`. In the docs it says:

> Removing Data  
Spark automatically monitors cache usage on each node and drops out old data partitions in a     least-recently-used (LRU) fashion. If you would like to manually remove an RDD instead of   waiting for it to fall out of the cache, use the RDD.unpersist() method.

From http://spark.apache.org/docs/latest/programming-guide.html#rdd-persistence

Am I reading that correctly, or do you think unpersisting is necessary? (Just so I know what's going on going forward)